src/patternfly/layouts/Level/level-vertical.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="pf-is-vertical {{ modifierClass }}">
```

Comment:
Can we remove this file, and use the modifierClass in the instance of the component?